tools/fog-local-network/fog_conformance_tests.py

```diff
@@ -899,8 +911,8 @@ def run(self, args):
 
         # Restarting the view server should not impact things.
         print("Restarting fog view server")
```

Comment:
:upside_down_face: Should this comment get updated?
```suggestion
        print("Restarting fog view store")
```